server/enterprise/metrics/metrics.go

```diff
@@ -1244,6 +1249,42 @@ func New(ps *platform.PlatformService, driver, dataSource string) *MetricsInterf
 	)
 	m.Registry.MustRegister(m.ClientGlobalThreadsLoadDuration)
 
+	m.MobileClientLoadDuration = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: MetricsNamespace,
+			Subsystem: MetricsSubsystemClientsMobileApp,
+			Name:      "mobile_load",
+			Help:      "Duration of the time taken from when a user opens the app and the app finally loads all relevant information (milliseconds)",
+			Buckets:   []float64{1, 1.5, 2, 3, 4, 4.5, 5, 5.5, 6, 7.5, 10, 20, 25, 30},
```

Comment:
What range do you expect for these values to be in? Because of how Prometheus handles histograms, any values above the maximum will be treated as if it was the maximum, so we should make sure that it's rare for this to be longer than 30ms. That value seems much lower than I'd expect even when the app is loading quickly

Reply:
Also, what's your reasoning behind these specific sets of buckets? At first glance, i t feels a bit arbitrary that these 3 use different bucket sizes

Reply:
These are seconds and not milliseconds :sweat_smile: So 30s is way over the acceptable time, so anything above that we don't care the specifics. It is too much.

My tests done locally (with development build, may be way off) showed me that load was around the 5s mark. That is why I added more precision around that value (0.5s increases). Similar idea with the other two, where I was taking values of around 500ms and 200ms. Since the expected results are different, the buckets also have to be different.

Reply:
Oh right. I forgot that these are in seconds when I said that they're in seconds in my other comment.

I also didn't think too much about how long these would take on mobile, so using custom buckets for these makes sense to me then